docs/en/data_types/nested_data_structures/simpleaggregatefunction.md

```diff
@@ -0,0 +1,39 @@
+# SimpleAggregateFunction(name, types\_of\_arguments…) {#data-type-simpleaggregatefunction}
+
+Unlike `AggregateFunction`:
```

Comment:
link to article about AggregateFunction + at least brief introduction about what it does for people who don't want to follow it